core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -276,6 +277,8 @@ object SparkEnv extends Logging {
       "spark.serializer", "org.apache.spark.serializer.JavaSerializer")
     logDebug(s"Using serializer: ${serializer.getClass}")
 
+    val serializerManager = new SerializerManager(serializer, conf)
```

Comment:
Yes, that's right. That would happen in the case where the user has configured `spark.serializer` to use Kryo.